xml/selinux.xml

```diff
@@ -450,54 +388,82 @@ setroubleshoot-server
-/lib/ld-linux.so.2              system_u:object_r:lib_t -&gt; system_u:object_r:ld_so_t</screen>
+Controlling terminal:           unconfined_u:object_r:user_tty_device_t:s0
+/etc/passwd                     system_u:object_r:passwd_file_t:s0
+/etc/shadow                     system_u:object_r:shadow_t:s0
+/bin/bash                       system_u:object_r:shell_exec_t:s0 -> system_u:object_r:shell_exec_t:s0
+/bin/login                      system_u:object_r:login_exec_t:s0
+/bin/sh                         system_u:object_r:bin_t:s0 -> system_u:object_r:shell_exec_t:s0
+/sbin/agetty                    system_u:object_r:getty_exec_t:s0
+/sbin/init                      system_u:object_r:bin_t:s0 -> system_u:object_r:init_exec_t:s0
+/usr/sbin/sshd                  system_u:object_r:sshd_exec_t:s0</screen>
   </example>
  </sect1>
 
- <sect1 xml:id="sec-selinux-configure">
-  <title>Configuring &selnx;</title>
+ <sect1 xml:id="sec-selinux-mode-enforcing">
+  <title>Putting &selnx; into enforcing mode</title>
   <para>
    At this point you have a completely functional &selnx; system and it is
    time to further configure it. In the current status, &selnx; is
    operational but not in enforcing mode. This means that it does not limit
    any activities, and it logs everything that it should be doing if it
    were in enforcing mode. Review the log files to learn what activities
-   are not allowed. As a first
-   test, put &selnx; in enforcing mode and find out if you can still use
-   your server after doing so: check that the option
-   <option>enforcing=1</option> is set in the &grub; configuration file,
-   while <option>SELINUX=enforcing</option> is set in
-   <filename>/etc/selinux/config</filename>. Reboot your server and see if
-   it still comes up the way you expect it to. If it does, leave it like
-   that and start modifying the server in a way that everything works as
-   expected. However, you may not even be able to boot the server properly.
-   In that case, switch back to the mode where &selnx; is not enforcing and
-   start tuning your server.
+   are not allowed.
   </para>
+  <para>After running &selnx; in permissive mode first, you can try to
+   put &selnx; in enforcing mode and find out if you can still use
+   your server after doing so.
+  </para>
+  <warning>
+   <title>Reset security context</title>
+   <para>
+    When systems run &selnx; in permissive mode, users and processes
+    might label various file-system objects incorrectly. This can cause
+    problems when switching to enforcing mode because &selnx; relies on
+    correct labels of file-system objects.
+   </para>
+   <para>
+    Before switching into enforcing mode, make sure to first reset the
+    security context (extended attributes). Do so by executing
+    <command>restorecon -R /</command> and then reboot the system.
+    System startup will take some time, as the entire file system
```

Comment:
If would recommend to remove this sentence because this is not true for SLES because we don't have the `SELinux-autorelabel` package.

The autorelabel would relabel the filesystem at boottime, if the `/.autorelabel` file is present. This would indeed take some time.

However we are doing a manual relabel using the `restorecon -R /` utility. This will also take some time but not at system startup but immediately.

Reply:
Thank you! I have removed the last sentence from the warning.